src/Microsoft.Azure.SignalR/ApplicationBuilderExtensions.cs

```diff
@@ -19,6 +19,9 @@ public static class AzureSignalRApplicationBuilderExtensions
         /// <param name="app">The <see cref="IApplicationBuilder"/>.</param>
         /// <param name="configure">A callback to configure the <see cref="ServiceRouteBuilder"/>.</param>
         /// <returns>The same instance of the <see cref="IApplicationBuilder"/> for chaining.</returns>
+#if NETCOREAPP3_1_OR_GREATER || NET5_0_OR_GREATER
```

Comment:
use `#if !NETSTANDARD2_0` to be consistent with how we support UseEndpoint in `AzureSignalRHostingStartup`?